cypress/e2e/notifications.spec.ts

```diff
@@ -0,0 +1,112 @@
+import * as api from "../api"
+import {
+  E2E_EMAIL_ADMIN,
+  E2E_EMAIL_COLLAB,
+  E2E_EMAIL_TEST_SITE,
+} from "../fixtures/constants"
+import {
+  genRandomString,
+  getNotificationsButton,
+  visitE2eEmailTestRepo,
+} from "../utils"
+
+const NEW_COMMENTS_BG_COLOR = "rgb(235, 248, 255)"
+const EMPTY_NOTIFICATIONS_TEXT = "There are no notifications for display."
+const getReviewRequestedNotifText = (email: string) =>
```

Comment:
nit: Feels weird that we are defining these constants here.. 
should we extract this out in our non-test codes, and then export those strings here? 
this way if we change the underlying string our tests will still pass

Reply:
sure - putting it in `NotificationsMenu.tsx` cos that's the only place where it's used.